modules/fun.js

```diff
@@ -0,0 +1,450 @@
+// @ts-check
+const Augur = require(`augurbot-ts`),
+  Discord = require(`discord.js`),
+  u = require(`../utils/utils`),
+  axios = require('axios'),
+  mineSweeperEmojis = { 0:'0⃣', 1:'1⃣', 2:'2⃣', 3:'3⃣', 4:'4⃣', 5:'5⃣', 6:'6⃣', 7:'7⃣', 8:'8⃣', 9:'9⃣', 10:'🔟', 'bomb':'💣' };
```

Comment:
This could be made an array, with index 9 being the bomb and removing 10. Requires a slight rework in the mapping function

```suggestion
  mineSweeperEmojis = ['0⃣', '1⃣', '2⃣', '3⃣', '4⃣', '5⃣', '6⃣', '7⃣', '8⃣', '💣'];
```